src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/RecapitiPersonaFisicaTest.java

```diff
@@ -698,6 +711,20 @@ public void nellaPaginaITuoiRecapitiSiControllaCheLIndirizzoPecNonSiaPresente()
         }
     }
 
+    @Then("Nella pagina I Tuoi Recapiti si controlla che l'indirizzo pec presente")
+    public void nellaPaginaITuoiRecapitiSiControllaCheLIndirizzoPecPresente() {
+        logger.info("Si controlla che la PEC non stata eliminata");
+
+        if (recapitiDestinatarioPage.siControllaPresenzaPEC()) {
+            logger.info("La PEC non è stata eliminata");
+        } else {
+            logger.error("La PEC è stata eliminata");
+            Assert.fail("La PEC è stata eliminata");
+        }
+    }
```

Comment:
Same behaviour of some method above, check if you can optimize this process and use the global webelement at the top of the file

Reply:
done

Reply:
This one is already present here:
https://github.com/pagopa/pn-frontend-e2e/blob/10524309fbd27d6c7d78103d86d876080fe07053/src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/RecapitiPersonaFisicaTest.java#L702-L712

Reply:
it is two different steps with, one is checking if pec eleminated, the other one checks that not deleted. I already did one optimization in siControllaPresenzaPec method